
uint32[] active_neuron_
float32 activation_level_
